styles/styles.css

```diff
@@ -548,6 +578,10 @@ footer {
     width: 57%;
   }
 
+  .headline > p {
```

Comment:
- [ ] Great job so far! 👏🏻  however according to the project requirement, 🤔  on the desktop size,  the closed button, and the menu mobile page must not be visible. Kindly correct this. You can use the `@media screen and (min-width: 768px) {`  to display none of your navigation mobile menu. 😃 

- Your design:

![18](https://user-images.githubusercontent.com/79058364/188523689-a3c7c5fd-a4a7-481c-9c95-462d44a1c659.png)

- And make sure that:

- [ ]  When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. 👍🏻 
- [ ]  When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears. 👍🏻